website/data/docs-nav-data.json

```
@@ -321,6 +321,19 @@
           }
         ]
       },
+      {
+        "title": "Cluster Peering",
+        "routes": [
+          {
+            "title": "Overview",
```

```suggestion
            "title": "What is Cluster Peering",
```
We should match the nav element to the page title so it's not so jarring for readers.